packages/fhir-converter/src/templates/cda/Utils/ContactRoleCode.hbs

```diff
@@ -0,0 +1,26 @@
+ "code":
+    {{#if (eq code "BILL")}}
```

Comment:
as a default, when writing these things, should we `toLower()` and `trim()` the string?

we don't want to fail on `bill ` or ` BILL    `

Reply:
I think trim makes sense. In this case, not sure if lower makes sense since it's a code and is really supposed to be capitalized. 

In the other case where we lowercased, it was because we were taking a free text value and trying to match it to something.

If you still think we should handle the lowercase condition, I do think that in the comparison file, the codes should look like they do in the docs (all caps) for readability and ease of reference. I can add a toUpper() function call before passing to this mapping so we maintain readability